common/src/main/java/com/wynntils/functions/generic/LogicFunctions.java

```diff
@@ -151,7 +151,7 @@ public FunctionArguments.RequiredArgumentBuilder getRequiredArgumentsBuilder() {
 
         @Override
         public List<String> getAliases() {
-            return List.of("mt");
+            return List.of("mt", "more_than", "gt");
```

Comment:
```suggestion
            return List.of("mt", "more_than", "gt",">");
```

or do function names have to be a-z?

Reply:
Also, just checking, is less than already `lt`?

Reply:
Technically not, but this looks really bad imo.

Reply:
> Also, just checking, is less than already `lt`?

yes